pages/api/mark/[id].js

```diff
@@ -0,0 +1,98 @@
+import { useSelector, useDispatch } from 'react-redux'
+import { markSquare, jumpTo } from '../../../redux/reducer'
+import { v4 as uuidv4 } from 'uuid';
+
+import sqlite from 'sqlite';
+
+export default async (req, res) => {
+  const {
+    query: { id },
+  } = req
+  // search into database the game id
+  
+  const gameId = req.body.gameId;
+  const db = await sqlite.open('./mydb.sqlite');
+  const game = await db.all('SELECT * FROM Game WHERE id=?',gameId);
```

Comment:
limit 1?

Reply:
Este comentario no lo entendí

Reply:
agregar a la sql query limit 1 asi no busca por mas de 1
https://stackoverflow.com/questions/28423687/is-it-necessary-to-use-limit-1-for-sql-select-for-performance